validator/project_validator.go

```diff
@@ -185,9 +183,7 @@ var projectSettingsValidators = []projectSettingsValidator{
 }
 
 // These validators have the potential to be very long, and may not be fully run unless specified.
-var longErrorValidators = []longValidator{
-	validateTaskSyncCommands,
-}
+var longErrorValidators = []longValidator{}
```

Comment:
We can just get rid of this and the --long flag, since there's nothing in here! (Can do in another PR but might be good to do as a quick-win in the nearterm just to keep things clean)

Reply:
Sure, I wasn't sure if we regularly cycle validators in here. I'll remove it in this PR!

Reply:
I've removed the long flag from validate command as well. The latest commit has the changes for the long flag/validator